pages/zh-cn/docs/index.mdx

```diff
@@ -9,9 +9,9 @@ labels:
 
 该网站上有几种类型的文档：
 
-- API 函数接口文档
-- ES6 特性
-- 上手指南
+-   API 函数接口文档
```

Comment:
Notice there's still a duplicated white space generated between "-" and the other characters.